components/brave_wallet_ui/common/trezor/trezor-bridge-transport.ts

```diff
@@ -21,13 +21,14 @@ export class TrezorBridgeTransport extends MessagingTransport {
 
   // T is response type, e.g. UnlockResponse. Resolves as `false` if transport error
   sendCommandToTrezorFrame = <T> (command: TrezorFrameCommand): Promise<T | false> => {
-    return new Promise<T>(async (resolve) => {
+    return new Promise<T | false>(async (resolve) => {
       let bridge = this.getBridge()
       if (!bridge) {
         bridge = await this.createBridge()
       }
       if (!bridge.contentWindow) {
-        return Promise.resolve(false)
+        resolve(false)
```

Comment:
@SergeyZhukovsky can utilize the same resolve function here rather than creating a new one from Promise.resolve

Reply:
I did and tslint alerted on it, maybe with eslint it will work better, will check when merged